src/Audio/SoundFont.purs

```diff
@@ -123,7 +122,7 @@ loadInstrument maybeLocalDir instrumentName = do
   res <- request $ defaultRequest
     { url = url, method = Left GET, responseFormat = ResponseFormat.string }
 
-  case res.body of
+  case res <#> _.body of
```

Comment:
I'm intrigued why you made this change.

Reply:
This is to accommodate the API change in Affjax. Previously `res.body` was an `Either`. Now `res` is an `Either` but `res.body` is just a `String`.

This particular line is equivalent to `_.body <$> res` or `map _.body res` or `map (\x -> x.body) res` or `mapFlipped res _.body` which is really what `<#>` is. An infix version of `mapFlipped`.

Reply:
Ah - of course.  Many thanks/